src/functionaltests/resources/features/status.feature

```diff
@@ -0,0 +1,19 @@
+
+@Status
+Feature: Test Status
+
+  @TestStatusIsOk
+  Scenario: Test Status Is AVAILABLE
+    Given "None" services is unavailable
```

Comment:
services --> service (since you only ever refer to a single service)

Reply:
Done.